src/bbmustache.erl

```diff
@@ -689,3 +707,73 @@ is_recursive_data(_)                                -> false.
 is_recursive_data([Tuple | _]) when is_tuple(Tuple) -> true;
 is_recursive_data(_)                                -> false.
 -endif.
+
+%% @doc Processes command-line commands (render, ...)
+-spec process_commands([getopt:option()], [string()]) -> ok.
+process_commands([], []) ->
+    help();
+process_commands([help | _], _) ->
+    help();
+process_commands([version | _], _Options) ->
+    {ok, AppConfig} = application:get_all_key(bbmustache),
+    Version = proplists:get_value(vsn, AppConfig),
+    io:format("~p~n", [Version]);
+process_commands([commands | _], _Options) ->
+    Commands = "render [file,...]",
+    io:format("~s~n", [Commands]);
+process_commands(Options, ["render" | TemplateFileNames]) ->
```

Comment:
When it no files,  I would like to make an error.

Reply:
f551d29a39d620da2886959b5731a2da1e5481c3